### PR TITLE
Get nvd3 from bower, delete from hqwebapp

### PR DESCRIPTION
**Pull request by orangejenny**
We already had nvd3 in bower, just weren't referencing it in custom reports.

@gcapalbo 
or @emord since you like reviewing custom reports

### CODE REVIEW

**Comment by orangejenny** [comment]
Also removing datatables-1.9, which isn't referenced anywhere

File: custom/icds_reports/templates/icds_reports/dashboard.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
               media="all"
               href="{% static 'font-awesome/css/font-awesome.css' %}"/>
 
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'hqwebapp/js/lib/nvd3/nv.d3.css' %}">
+        <link type="text/css" rel="stylesheet" media="all" href="{% static 'nvd3/src/nv.d3.css' %}">
```
Comment (emord) [inline custom/icds_reports/templates/icds_reports/dashboard.html]: Seems weird to have the css here without any JS. Are we sure this is actually used? Same for the other refs here that have this without 

FWIW I blocked this on ICDS with ublock and it seemed fine,but i'm not sure if i'm missing something

**Review by emord — APPROVED** [review]
> since you like reviewing custom reports

-_-

File: custom/icds_reports/templates/icds_reports/dashboard.html (hunk 1 above)
Comment (orangejenny) [reply custom/icds_reports/templates/icds_reports/dashboard.html]: The templates that only have the css path edited do use the js, but they're already pulling it from bower_components. I'd guess that whoever nvd3 moved to bower only grepped for the js paths.

Comment (emord) [reply custom/icds_reports/templates/icds_reports/dashboard.html]: Ah that makes sense

**Pull request merged**